packages/react-shortcuts/src/Shortcut/hooks.ts

```diff
@@ -11,23 +17,32 @@ export interface Options {
   held?: HeldKey;
   node?: HTMLElement | null;
   ignoreInput?: boolean;
+  acceptedDefaultIgnoredTags?: DefaultIgnoredTag[];
   allowDefault?: boolean;
 }
 
+const DEFAULT_IGNORED_TAGS = [
```

Comment:
Honestly I'd probably just go with a `as const` declaration on this instead of the enum, and then export this value. That way we don't need to worry about adding something to the enum but missing it here, and we'll still get the typing benefits by doing `typeof DEFAULT_IGNORED_TAGS[number]` for the option type definition

Reply:
Good call 👍